### ISSUE TITLE
Test card timer should have more accurate label

### ISSUE THREAD

**Issue by nathancrtr** [CDCgov/prime-simplereport#4046]
## Area or Component
Conduct Test

## Quick Summary
The timer's aria name is "15:00", but would be more accurately labelled "timer" (or at least given a role of timer)

## Accessibility Violation
* `aria-name-missing-incorrect`
    * The element's accessible name is missing or is not appropriate for the element's purpose

## Action Requested
The test card timer should have a more descriptive aria label: "timer" should suffice.

Can also consider adding a `role` of timer

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/main/frontend/src/app/testQueue/TestTimer.tsx#L234-L275

**Comment by mehansen** [comment]
I wonder if these buttons should have labels that explain that you can click to start and reset the timers, since without the visible symbols it's kind of hard to tell what they do

@vz3
